launchable/commands/stats/test_sessions.py

```diff
@@ -0,0 +1,58 @@
+import click
+import os
+from typing import List
+
+from ...utils.click import KeyValueType
+from ...utils.env_keys import REPORT_ERROR_KEY
+from ...utils.flavor import normalize_flavors
+from ...utils.http_client import LaunchableClient
+
+
+@click.command()
+@click.option(
+    '--days',
+    'days',
+    help='How many days of test sessions in the past to be stat',
+    type=int,
+    default=7
+)
+@click.option(
+    "--flavor",
+    "flavor",
+    help='flavors',
+    cls=KeyValueType,
+    multiple=True,
+)
+@click.pass_context
+def test_sessions(
+    context: click.core.Context,
+    days: int,
+    flavor: List[str] = [],
+):
+    # We originally wanted to write it like `params: Dict[str, Any]`, but python 3.5 does not support it,
+    # so we gave an empty list in the 'flavor' key to give a type hint.
+    # If we don't write this, `pip run type` will assume it is Dict[str, int], and the check will not pass.
+    params = {'days': days, 'flavor': []}
+    flavors = []
+    for f in normalize_flavors(flavor):
+        flavors.append('%s=%s' % (f[0], f[1]))
+
+    if flavors:
+        params['flavor'] = flavors
+    else:
+        del params['flavor']
```

Comment:
[nits]
Just to avoid removing a key from dict does not raise error for the key does not exist (I know obviously it does not happen in this code), I prefer to write `params.pop("flavor", None)` which will remove a key if exists or do nothing if not exists.

Reply:
I see. This is a very good lesson. Thank you very much.